operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -36,8 +36,16 @@ type ImageContentSourcePolicySpec struct {
 	// For example, if policy A has mirrors `a, b, c` and policy B has mirrors `c, d, e`, the
 	// mirrors will be used in the order `a, b, c, d, e`.  If the orders of mirror entries conflict
 	// (e.g. `a, b` vs. `b, a`) the configuration is not rejected but the resulting order is unspecified.
+	//
+	// Deprecated: RepositoryDigestMirrors will be deprecated. Use RepositoryMirrors satisfies RepositoryDigestMirrors
```

Comment:
will or are deprecated? And what does it mean?

Also, this is an alpha API. Can you point me to a doc for the promotion to beta?

Reply:
This comment will let godoc hide the api.
What doc is needed? There's discussion in the enhancement plans to drop this API after upgrade to alpha2 and it is pointed out in the enhancement.   https://github.com/openshift/enhancements/pull/690/files#diff-8a9e721e9e6cf20d40bec3e5bc4a84d8a5c8acebb3d95a3bafbd9e50e7d47933R165

Reply:
Added the link to the comment https://github.com/openshift/enhancements/pull/690/files#diff-8a9e721e9e6cf20d40bec3e5bc4a84d8a5c8acebb3d95a3bafbd9e50e7d47933R165

Reply:
you are aware that graceful dropping of fields with CRDs doesn't work if you have no conversion webhook? Do you plan a conversion webhook?

Reply:
@sttts  Yes, a conversion webhook is planned. Added a enhancement doc https://github.com/openshift/enhancements/pull/863

Reply:
I think this comment still holds.